alica_tests/src/test/test_blackboard.cpp

```diff
@@ -65,6 +65,7 @@ TEST_F(TestBlackBoard, testJsonBlackboardBehavior)
 TEST_F(TestBlackBoard, testNotInheritBlackboardFlag)
 {
     ae->start();
+    ae->getAlicaClock().sleep(alica::AlicaTime::milliseconds(100));
```

Comment:
Maybe use STEP_UNTIL

Reply:
Engine isn't running in step mode in these tests. Out of scope of this pr.

Reply:
Would be RUN_UNTIL then